jekyll/_includes/global-nav.html

```diff
@@ -59,5 +59,12 @@
         </div>
       </nav>
     </div>
+
+    <!-- Experiment:   // https://app.optimizely.com/v2/projects/16812830475/experiments/20616121296/variations -->
+    <div class="dd-global-nav--links" style="display: none">
+      <a class="link home dd-devhub-docs-nav-link" href="/docs/2.0">Home</a>
+      <a class="link dd-devhub-docs-nav-link" href="https://circleci.com/developer/images">Images</a>
+      <a class="link dd-devhub-docs-nav-link" href="https://circleci.com/developer/orbs">Orbs</a>
```

Comment:
Doesn't seems like you use the `dd-devhub-docs-nav-link` class for styling so I would suggest removing it to not clutter the page unnecessarily. When it comes to your event tracking you could just do  `$(".dd-global-nav--links a")`